tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
+    "module": "commonjs",
+    "target": "es6",
```

Comment:
I have changed the target to ES5, here is the commit: https://github.com/alex3165/react-mapbox-gl/pull/114/commits/ec97fb744f3555de9d2e8bd4482816af6ecd73ea